protocol/daemons/pricefeed/client/price_fetcher/price_fetcher.go

```diff
@@ -183,6 +184,27 @@ func (pf *PriceFetcher) RunTaskLoop(requestHandler daemontypes.RequestHandler) {
 	}
 }
 
+// logMarketAvailability emits telemetry that tracks whether a market was available when queried on an exchange.
+// Success is tracked by (market, exchange) so that we can track the availability of each market on each exchange.
+func logMarketAvailability(exchangeId types.ExchangeId, id types.MarketId, available bool) {
```

Comment:
Nit: `emitMarketAvailabilityMetrics` or something to better reflect the intention of this function?